src/Components/TodoList.jsx

```diff
@@ -44,6 +49,14 @@ const TodoList = ({ tasks, getTasksFromFirebase }) => {
                           getTasksFromFirebase={getTasksFromFirebase}
                         />
                       </div>
+                      <button
+                        className="danger-button button-base"
```

Comment:
You dont't need to use the button classNames. Instead use the Button Component and and pass `type="danger"`, `btnText="Delete"`